packages/react-devtools-shared/src/backend/views/TraceUpdates/canvas.js

```diff
@@ -191,7 +209,13 @@ function destroyNative(agent: Agent) {
 
 function destroyWeb() {
   if (canvas !== null) {
+    // $FlowFixMe[prop-missing]: Flow doesn't recognize Popover API
+    // $FlowFixMe[incompatible-use]: Flow doesn't recognize Popover API
+    canvas.hidePopover();
```

Comment:
I think you should check here if popover is opened to avoid getting InvalidStateException